src/components/HotspotConfigurationPicker.tsx

```diff
@@ -15,6 +15,37 @@ import { decimalSeparator, groupSeparator, locale } from '../utils/i18n'
 import { useColors } from '../theme/themeHooks'
 import { Antenna, Antennas } from '../constants/antennas'
 
+export type AntennaId =
```

Comment:
You need to remove these lines, 18 - 48.

They got moved to the `constants/antennas` file